db/migrate/20150508141500_change_name_display_type_in_communities.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeNameDisplayTypeInCommunities < ActiveRecord::Migration
+  def up
+    execute("UPDATE communities
+      SET name_display_type='full_name'
+      WHERE name_display_type = '';
```

Comment:
Do we also need to handle the case where this is NULL, or does this also cover that already?